src/lwc/geFormField/geFormField.js

```diff
@@ -14,6 +14,9 @@ const DELAY = 300;
 const RICH_TEXT_FORMATS = [
     'font', 'size', 'bold', 'italic', 'underline', 'strike', 'list', 'indent', 'align', 'link', 'clean', 'table', 'header'
 ];
+const CURRENCY = 'currency';
```

Comment:
Similar to the granularity() function, it would be great not to have to duplicate the constants